src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -23,6 +23,14 @@
 /** A candidate for a public office position that will appear on voter ballots */
 @AutoValue
 public abstract class Candidate {
+  public static final String ENTITY_NAME = "Candidate";
```

Comment:
maybe ENTITY_TYPE or ENTITY_KIND.

I think "kind" is the nomenclature that Datastore documentation uses.
https://cloud.google.com/datastore/docs/concepts/entities#kinds_and_identifiers